Source/JavaScriptCore/runtime/VM.h

```diff
@@ -1007,6 +1007,13 @@ class VM : public ThreadSafeRefCounted<VM>, public DoublyLinkedListNode<VM> {
     template<typename Func>
     void forEachDebugger(const Func&);
 
+    void changeNumberOfActiveJITPlans(int64_t value)
+    {
+        m_numberOfActiveJITPlans += value;
+    }
+
+    int64_t numberOfActiveJITPlans() const { return m_numberOfActiveJITPlans.load(); }
```

Comment:
Do these really need to be sequentially consistent? It seems like these could be `std::memory_order_relaxed`.

Reply:
Yeah, I think relaxed is enough. Changed.